docs.bruno/Update req. (PUT).bru

```diff
@@ -0,0 +1,11 @@
+meta {
+  name: Update req. (PUT)
+  type: http
+  seq: 3
+}
+
+put {
+  url: 
```

Comment:
The URL and body for the PUT request are missing. Please provide the correct URL and an example body for the PUT request in the Bruno file. [important]